README.md

```diff
@@ -743,6 +743,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Emi Gomez](https://github.com/emigl)
   - [Emmanuel Engelbert](https://github.com/EmmanuelEngelbert)
   - [Emmanuel Obiechina](https://github.com/chibuike-19)
+  - [Emily Marie Ahtúnan](https://github.com/Emmarie-Ahtunan)
```

Comment:
Hey Emily, I think you need to insert your name on line 744 after the name 'Emi Gomez' according to the alphabetical order. Please do let us know if you need any help with it 🙂